style.css

```diff
@@ -0,0 +1,635 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+html,
+body {
+  overflow-x: hidden;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main-page {
+  background-color: lightgray;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  margin: 0 30px;
+  gap: 40%;
+}
+
+.top {
+  display: none;
+  list-style: none;
+  gap: 20px;
+  justify-content: flex-end;
+  padding: 10px 20px;
+}
+
+.lg-logo {
+  display: none;
+}
+
+.logo {
+  padding: 10px;
+  font-weight: 700;
+  display: none;
+}
```

Comment:
- [ ] Derar @monicah79, good job making some of the changes. Although as the previous reviewer mention your design is not the same as the original design Please Change the spacing, font size, the font family for the header.
## Figma design
![image](https://user-images.githubusercontent.com/69077061/204107201-78e945f9-9117-4373-a079-03fefe2ebc1e.png)

## Your design 
![image](https://user-images.githubusercontent.com/69077061/204107228-b2b70300-892c-4f6b-84ef-cf8ba26ce48a.png)